### DISCUSSION TITLE
Shortcut to switch to preview-tui window

### DISCUSSION THREAD

**Discussion by Praful** [jarun/nnn#1915] [General]
Is there a shortcut key to switch between the main nnn window and the preview window for preview-tui plugin? If not, how can I create one?

I was hoping the Vim shortcuts `CTRL-W+J/L/K/H` would work to switch to an adjacent window.

**Comment by luukvbaal** [comment]
That totally depends on the way you're using preview-tui. For tmux, https://unix.stackexchange.com/a/253324 seems relevant.

**Comment by Praful** [comment]
Thanks - I didn't realise it was a function of the terminal. I'm using kitty. For reference: 

- Switch to the next window: Press `Ctrl + Shift + ]`.
- Switch to the previous window: Press `Ctrl + Shift + [`.
- Move to a specific window: Press `Ctrl + Shift + <window number>` (where <window number> is 1, 2, 3, etc.).
